aepp/destinationinstanceservice.py

```diff
@@ -0,0 +1,81 @@
+# Internal Library
+import aepp
+from aepp import connector
+from typing import Union
+import logging
+from .configs import ConnectObject
+
+class Instance:
```

Comment:
to keep things consistent with the filename, should we rename this class to `DestinationInstanceService` instead?